npm/ng-packs/packages/theme-shared/src/lib/utils/date-parser-formatter.ts

```diff
@@ -13,9 +13,9 @@ function toInteger(value: any): number {
 
 @Injectable()
 export class DateParserFormatter extends NgbDateParserFormatter {
-  constructor(private configState: ConfigStateService, @Inject(LOCALE_ID) private locale: string) {
-    super();
-  }
+  private configState = inject(ConfigStateService);
```

Comment:
We can keep this super call here just in case. This is because we extend an external formatter called `NgbDateParserFormatter`